lib/iris/tests/test_basic_maths.py

```diff
@@ -632,7 +634,7 @@ def test_log2(self):
 
     def test_log10(self):
         e = iris.analysis.maths.log10(self.cube)
-        self.assertCMLApproxData(e, ('analysis', 'log10.cml'))
+        self.assertCMLApproxData(e, ('analysis', 'log10.cml'), rtol=1e-6))
```

Comment:
Need this for the `Cray` ... don't think it's worth digging to discover - maybe platform or `numpy` build related.

It's a singlton glitch ...